tools/api-markdown-documenter/.eslintrc.cjs

```diff
@@ -9,6 +9,9 @@ module.exports = {
 		project: ["./tsconfig.json"],
 	},
 	rules: {
+		// Too many false positives with array access
+		"@fluid-internal/fluid/no-unchecked-record-access": "off",
```

Comment:
It isn't clear to me if this is by design or an oversight, but this rule requires at any usage of array contents needs to assume the value can be `undefined`, even when the boundaries have already been checked. This seems to me to be assuming code is not properly leveraging the type system.